### PR TITLE
fixes Git A-132

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
- fixes Git A-132

## Output
Validate-Mediator-WSO2-Enterprise-Integrator-Documentation

### CODE REVIEW

**Review by rosensilva — APPROVED** [review]

File: en/micro-integrator/docs/references/mediators/validate-Mediator.md [hunk 1]
```diff
@@ -299,9 +299,11 @@ to be sent back to the party, which sends the message.
     <schema key="conf:/schema/StockQuoteSchema.json"/>
     <on-fail>
         <payloadFactory media-type="json">
-            <format>{"Error":"$1"}</format>
+            <format>{"Error":"$1",
+            "Error Details " : " $2"       }</format>
```
Comment (arunans23) [inline en/micro-integrator/docs/references/mediators/validate-Mediator.md:303]:
```suggestion
            "Error Details" : "$2"       }</format>
```
Shall we remove the unncessary spaces?

File: en/micro-integrator/docs/references/mediators/validate-Mediator.md [hunk 2]
```diff
@@ -322,6 +324,16 @@ An example for a valid JSON payload request is given below.
 }
 ```
 
+When you send an invalid JSON, the following response will appear.
+
+```
+{
+    "Error": "object has missing required properties ([\"request\"])",
+    "Error Details ": " Error while validating Json message error: object has missing required properties ([\"request\"])\n    level: \"error\"\n    schema: {\"loadingURI\":\"#\",\"pointer\":\"/properties/getQuote\"}\n    instance: {\"pointer\":\"/getQuote\"}\n    domain: \"validation\"\n    keyword: \"required\"\n    required: [\"request\"]\n    missing: [\"request\"]\n"
```
Comment (arunans23) [inline en/micro-integrator/docs/references/mediators/validate-Mediator.md:332]:
```suggestion
    "Error Details": "Error while validating Json message error: object has missing required properties ([\"request\"])\n    level: \"error\"\n    schema: {\"loadingURI\":\"#\",\"pointer\":\"/properties/getQuote\"}\n    instance: {\"pointer\":\"/getQuote\"}\n    domain: \"validation\"\n    keyword: \"required\"\n    required: [\"request\"]\n    missing: [\"request\"]\n"
```

File: en/micro-integrator/docs/references/mediators/validate-Mediator.md [hunk 3]
```diff
@@ -333,9 +345,11 @@ schema.
     <schema key="conf:/schema/StockQuoteSchema.json"/>
     <on-fail>
         <payloadFactory media-type="json">
-            <format>{"Error":$1"}</format>
+            <format>{"Error":"$1",
+            "Error Details " : " $2"       }</format>
```
Comment (arunans23) [inline en/micro-integrator/docs/references/mediators/validate-Mediator.md:349]:
```suggestion
            "Error Details" : "$2"       }</format>
```

File: en/micro-integrator/docs/references/mediators/validate-Mediator.md [hunk 4]
```diff
@@ -357,3 +371,12 @@ An example for a valid JSON request payload is given below.
    }
 }
 ```
+
+When you send an invalid JSON, the following response will appear.
+
+```
+{
+    "Error": "object has missing required properties ([\"request\"])",
+    "Error Details ": " Error while validating Json message error: object has missing required properties ([\"request\"])\n    level: \"error\"\n    schema: {\"loadingURI\":\"#\",\"pointer\":\"/properties/getQuote\"}\n    instance: {\"pointer\":\"/getQuote\"}\n    domain: \"validation\"\n    keyword: \"required\"\n    required: [\"request\"]\n    missing: [\"request\"]\n"
```
Comment (arunans23) [inline en/micro-integrator/docs/references/mediators/validate-Mediator.md:380]:
```suggestion
    "Error Details": "Error while validating Json message error: object has missing required properties ([\"request\"])\n    level: \"error\"\n    schema: {\"loadingURI\":\"#\",\"pointer\":\"/properties/getQuote\"}\n    instance: {\"pointer\":\"/getQuote\"}\n    domain: \"validation\"\n    keyword: \"required\"\n    required: [\"request\"]\n    missing: [\"request\"]\n"
```

**Comment by dinuka10** [comment]
Hi,

Fix is correct

Thanks and Regards,
Dinuka

**Pull request merged**